src/System.Text.Encoding/tests/Fallback/EncoderReplacementFallbackTests.cs

```diff
@@ -109,7 +109,8 @@ public void CreateFallbackBuffer_Fallback_InvalidSurrogateChars_ThrowsArgumentOu
             EncoderFallbackBuffer buffer = new EncoderReplacementFallback().CreateFallbackBuffer();
 
             Assert.Throws<ArgumentOutOfRangeException>("charUnknownHigh", () => buffer.Fallback('a', '\uDC00', 0));
-            Assert.Throws<ArgumentOutOfRangeException>("CharUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
+            Assert.Throws<ArgumentOutOfRangeException>(
+                RuntimeDetection.IsNetFramework ? "CharUnknownLow" : "charUnknownLow", () => buffer.Fallback('\uD800', 'a', 0));
```

Comment:
@tarekgh We went this PR to be open, ready to be merged together with the matching CoreCLR changes. The matching CoreCLR changes won't pass the CI otherwise.